miden-crypto/src/rand/rpx.rs

```diff
@@ -167,11 +160,15 @@ impl RandomCoin for RpxRandomCoin {
             }
         }
 
-        if values.len() < num_values {
-            return Err(RandomCoinError::FailedToDrawIntegers(num_values, values.len(), 1000));
-        }
+        assert_eq!(
```

Comment:
Should we return an error for API compatibility? This does seem unlikely so I'm fine with the assert

Reply:
Here and elsewhere we need an answer to [this comment](https://github.com/0xMiden/crypto/pull/720#discussion_r2633849941). Happy to remove though.